.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "git.ignoreLimitWarning": true
+}
```

Comment:
- [ ] [OPTIONAL] You can add this file to your gitignore as it is simply a personal vscode settings file and does not need to be tracked in your repo.